EventsExpress/ClientApp/src/components/event/event-form.js

```diff
@@ -69,6 +74,7 @@ class EventForm extends Component {
                         crop={true}
                         cropShape='rect'
                         loadImage={() => photoService.getFullEventPhoto(this.props.eventId)}
+                        uploadImage={this.saveImage(this.props.eventId)}
```

Comment:
I believe `uploadImage` is not used anywhere. Do we want to remove it?

Reply:
Sure, i delete this line and saveImage method